src/main/java/org/galatea/starter/domain/HistoricalPriceDB.java

```diff
@@ -0,0 +1,109 @@
+package org.galatea.starter.domain;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import lombok.Builder;
+
+
+@Entity
+@IdClass(CompositePrimaryKey.class)
+public class HistoricalPriceDB {
+
+  private BigDecimal close;
+  private BigDecimal high;
+  private BigDecimal low;
+  private BigDecimal open;
+  @Id
+  private String symbol;
+  private Integer volume;
+  @Id
+  private LocalDate date;
+  @Id
+  private LocalTime minute;
+  private LocalTime timestamp;
+
+  public HistoricalPriceDB() {
+  }
+
+  public HistoricalPriceDB(BigDecimal close, BigDecimal high, BigDecimal low,
+      BigDecimal open, String symbol, Integer volume, LocalDate date, LocalTime minute,
+      LocalTime timestamp) {
+    this.close = close;
+    this.high = high;
+    this.low = low;
+    this.open = open;
+    this.symbol = symbol;
+    this.volume = volume;
+    this.date = date;
+    this.minute = minute;
+    this.timestamp = timestamp;
+  }
+
+  public BigDecimal getClose() {
```

Comment:
you should not have physical getters and setters. look at lombok's @Getter and @Setter, and @Data

Reply:
This should be resolved now.